tensorrt_llm/_torch/auto_deploy/export/export.py

```
@@ -266,11 +273,18 @@ def _expand_moe_experts_in_graph(
         if not is_op(node, moe_ops):
             continue
 
-        # Collect indices of list-of-node arguments (expert weight/scale lists)
+        # Collect indices of List[Tensor] arguments from the op schema – these
```

Ruff (0.15.1)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)